cmd/list.go

```diff
@@ -40,6 +41,10 @@ func newListCommand(opts *dobiOptions) *cobra.Command {
 	flags.BoolVarP(
 		&listOpts.all, "all", "a", false,
 		"List all resources, including those without descriptions")
+	flags.BoolVarP(
+		&listOpts.groups, "groups", "g", false,
+		"List resources sorted by their matching tags. Only resources"+
+			"with configured tags will be listed.")
```

Comment:
I think there is a space missing here at the end of the first line.

```suggestion
		"List resources grouped by by tag. Only resources "+
			"with configured tags will be listed.")
```

Reply:
Oh you are right, the space is missing. I also will use your text.